scripts/flush.py

```diff
@@ -41,15 +41,14 @@ def main(collection_ex_id):
 
     samples = [sample['id'] for sample in sample_units]
```

Comment:
```suggestion
     samples = [sample['id'] for sample in sample_units]
     print(f'Sample units for collection exercise: {len(samples)}')
```

Reply:
Added in along with the other suggestions 👍

I'd expect 404 where multiple IACs are used for a case, the first request with an IAC will be used to flush the session from EQ then the following requests will return 404s as that case will have been flushed. Dave was speaking to EQ last week about EQ not starting a new session for each IAC (not sure if this is the case in preprod/prod).